setup.py

```diff
@@ -25,7 +25,6 @@
         'License :: OSI Approved :: BSD License',
         'Programming Language :: Python',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
```

Comment:
Not sure if this belongs in the current PR but we also need to add a specifier for 3.9
```suggestion
        'Programming Language :: Python :: 3.8',
        'Programming Language :: Python :: 3.9',
```

Reply:
This change has nothing to do with dropping support for Python 3.5. Honestly, I'd rather just remove these trove classifiers as they just add toil to my life.

Reply:
@jaraco I recently learned that Poetry generates these automatically from the value of `python_requires`. I wonder if setuptools could do the same.

Reply:
I also just remembered that CherryPy docs mention somewhere the min and the max supported versions that are retrieved from these specifiers: https://github.com/cherrypy/cherrypy/blob/main/docs/conf.py#L32-L41. So we need to keep them up-to-date at least for now.

Reply:
Definitely +1 for Setuptools to generate the classifiers automatically.